docs/GettingStartedDocs/Contributors/building_oe_sdk.md

```diff
@@ -31,11 +31,9 @@ Please refer to the following [documentation](/docs/GettingStartedDocs/SGXSuppor
    | SGX1                       | SGX1 or SGX1+FLC                    |
    | Simulation                  | Any level                           |
 
-   On Linux, if your target system does not have any SGX hardware support, you may want to choose simulation.
+   On both Linux and Windows, if your target system does not have any SGX hardware support, you can leverage the simulation mode to use the SDK.
```

Comment:
I would just cut this sentence entirely since the previous section already describes simulation mode and its use; lines 34-36 used to be just caveats which are no longer relevant.